ecs-logs-collector.sh

```diff
@@ -496,6 +499,58 @@ get_docker_containers_info() {
   ok
 }
 
+get_docker_sysconfig() {
+  try "Gathering Docker sysconfig"
+
+  if [ -e /etc/sysconfig/docker ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker "${info_system}"/docker/sysconfig-docker
+    ok
+  else
+    info "/etc/sysconfig/docker not found"
+  fi
+
+ try "Gathering Docker storage sysconfig"
+
+  if [ -e /etc/sysconfig/docker-storage ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/sysconfig/docker-storage "${info_system}"/docker/sysconfig-docker-storage
+    ok
+  else
+    info "/etc/sysconfig/docker-storage not found"
+  fi
+}
+
+
+get_docker_daemon_json(){
+  try "Gathering Docker daemon.json"
+
+  if [ -e /etc/docker/daemon.json ]; then
+    mkdir -p "${info_system}"/docker
+    cp /etc/docker/daemon.json "${info_system}"/docker/daemon.json
+    ok
+  else
+    info "/etc/docker/daemon.json not found"
+  fi
+}
+
+get_docker_systemd_config(){
+  try "Gathering Docker systemd unit file"
+  
+  mkdir -p "${info_system}"/docker
+  if systemctl cat docker.service > "${info_system}"/docker/docker.service 2>/dev/null; then
```

Comment:
It's worth noting that the unit file will be empty (as in `touch "$info_system/docker/docker.service`) even when the system isn't running systemd or if there isn't a `docker.service` file.

This can be addressed in the `else` (for both `docker.service` and `containerd.service`):

```bash
if systemctl cat docker.service > "${info_system}"/docker/docker.service 2>/dev/null; then
  ok
else
  rm -f "$info_system/docker/docker.service"
  warning "docker.service is not present"
fi
```

Let's remove the potential for confusion by adding this. This keeps things simple for folks quickly paging over the list of files in the resulting archive. 

I'll click "approve" now, howevere I'd really like to see this one last check handled to keep the right context for deep dive debugging.